fe/src/project/AspNetCore/areas/News/components/NewsCatalog/NewsCatalog.scss

```diff
@@ -0,0 +1,90 @@
+.news-catalog {
+
+    .image img {
+        border-radius: 0.5rem;
+    }
+
+    .hero-news {
+        cursor: pointer;
+    }
+
+    .data-text {
+        margin-left: 0.5rem;
+    }
+
+    .news-data {
+        display: flex;
+        align-items: center;
+        margin: 0.5rem 0;
+
+        .data {
+            font-size: 0.8rem;
+            display: flex;
+            align-items: center;
+
+            svg {
+                fill: $primary;
+            }
+
+            &:first-child {
+                margin-right: 1rem;
+            }
+        }
+    }
+
+    .news-title {
```

Comment:
**suggestion:** Let's get rid of it completely. Use standard title and subtitle of Bulma: https://bulma.io/documentation/elements/title/